### PR TITLE
Issue #1241 Use 'make prerelease' target

### LINKED ISSUE #1241: Use 'make prerelease' target
When setting up the build definition for GitLab, I noticed from the error reporting that we actually use an unnecessary target for make in most of the CI:

  `make test cross fmtcheck prerelease`, while `make prerelease` (#1240) would have been enough.

Seems that this output is shallowed by all of the CI environments

### PR DESCRIPTION
**Pull request by gbraad**
Fixes #1241

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ install:
 - export PATH=$PATH:/tmp/glide/${GLIDE_OS_ARCH}
 
 script:
-- make cross test fmtcheck prerelease
+- make prerelease
```
Comment (praveenkumar) [inline .travis.yml]: LGTM 👍

**Comment by gbraad** [comment]
if @budhrg is OK with this, we can merge

**Comment by budhrg** [comment]
Yes, it is Ok. We updated the dependencies of `make prerelease` to have most of the checks to unburden `release` target. It looks more cleaner now. 

LGTM as well.

**Comment by budhrg** [comment]
Merged and thanks

**Pull request merged**